_posts/2020-01-24-my-favorite-artists.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "My Favorite Artists And You Can Too!"
+date: 2020-01-24 21:02:00 -0500
+category: blog
+author: Aaron
+author_twitter: shamshirz
+image: /assets/spotify.png
+---
+
+Sharing my recent Spotify 🎧 favorites with Elm and AWS
+![Spotify Logo]({{ "/assets/spotify/spotify.png" | relative_url }}){:class="excerpt-image"}
+
+
+<!--more-->
+I love Spotify. My music taste can often give a better summary of myself than I can. Conveniently, they also have a nice API that can make sharing that with your friends pretty easy too 😏
+
+This year a friend and I shared past playlists as a way to catch up and realized they were a great way to journal. The best way to share that time and experience with each other was to listen to the playlists on our own and at our own pace. This was a surprising, no-pressure way to share myself and I wanted to explore it even more.
+
+## Live Sharing
+
+How can I share a snapshot of myself, without it becoming dated, and without it being an overwhelming task? So far, I had the core data about me - Spotify - and the idea that I wanted it to stay current. There is a very suitable API for the task that will give a list of your most listened to artists over a time range. I found this to be a great starting point. I broke down my wild ideas into a project scoped for a reasonable amount of time and ended up with a serverless API to fetch my top Spotify artists plus a UI widget for the blog to display it 👇. [See how on our Github](https://github.com/SylverStudios/spotify-highlights)
```

Comment:
here and in the title we got these boxes - i think some unicode translation went wrong somewhere

Reply:
dang, they look good on MacOS Chrome and Firefox…I was trying to add emojis 😢 I'll try to swap em

Reply:
Maybe it's a Linux thing? Looks good for me on Chrome Windows.